_i18n/en.yml

```diff
@@ -174,6 +174,9 @@ team:
   mrw: Malware Response Workgroup
   mrw_descr: The Malware Response Workgroup provides resources and live support for multiple types of Monero-related malware.
   mrw_start: Most of the activity of this workgroup happens on the IRC chat. If you wish to contribute, just join the channel to ask questions or provide support.
+  mpwg: Monero Policy Working Group
```

Comment:
I would keep the naming consistent with the other workgroups (e.g. Monero Policy Workgroup), but if you prefer 'working group', fine for me.

Reply:
Not sure why this name was chosen, but the chosen name is as-is.

Reply:
I didn't understand it's the actual name of the workgroup, since they call it "Monero Policy Workgroup" elsewhere (for example https://github.com/monero-project/meta/issues/537), but whatever they prefer.

Reply:
@erciccione we are going with working group